tests/008-test-default-lv-data-size.sh

```diff
@@ -0,0 +1,84 @@
+source $SRCDIR/libtest.sh
+
+# Default size of LV that is created from the volume group 
+# by docker-storage-setup is 40% of the free space of the
+# VG, i.e DATA_SIZE=40%FREE. If dss gets modified and changes
+# this default behavior, below code should be modified to 
+# reflect changes
+test_default_lv_size() {
+  local devs=$TEST_DEVS # devices set by user in dss-config
+  local test_status
+  local testname=`basename "$0"`
+  local vg_name
+  local vg_name="dss-test-foo"
+  local pe_total
+  local pe_free
+  local pe_used_percentage
+  
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  for vg in $(vgs --noheadings -o vg_name); do
+    if [ "$vg" == "$vg_name" ]; then
+      echo "ERROR: $testname: Volume group $vg_name already exists."
+      return 1
+    fi
+  done
+
```

Comment:
This is copy paste in all tests. Can we introduce a helper function in libtest and make all tests call that helper function.